.gitignore

```diff
@@ -4,3 +4,4 @@ bower_components
 node_modules
 yarn-error.log
 yarn.lock
+package-lock.json
```

Comment:
`package-lock.json` is intended to be checked into source control. You should commit this file. According to npm help package-lock.json

https://github.com/npm/cli/blob/release-6.14.7/docs/content/configuring-npm/package-lock-json.md

Reply:
Ok, the file was added on the way